src/object/cli_ec.c

```diff
@@ -2810,6 +2810,9 @@ obj_recx_ec2_daos(struct daos_oclass_attr *oca, int shard,
 			 cell_nr;
 	}
 
+	if (total == 0)
+		goto out;
```

Comment:
just curious that for array type, is it reasonable to get zero iod_nr (or zero rx_nr for all recxs)?

Reply:
if rx_nr ==0, then it is possible.

Reply:
> if rx_nr ==0, then it is possible, since it is from remote server, just adding this check. maybe should add some warning message here.